packages/portfolio-contract/src/evm-account-kit.exo.ts

```diff
@@ -0,0 +1,265 @@
+/**
+ * @file Durable kit managing EVM account provisioning per Axelar chain.
+ */
+import { makeTracer } from '@agoric/internal';
+import { Fail } from '@endo/errors';
+import { M } from '@endo/patterns';
+import { VowShape } from '@agoric/vow';
+import type { MapStore } from '@agoric/store';
+import type { Vow, VowKit, VowTools } from '@agoric/vow';
+import type { Zone } from '@agoric/zone';
+import type { AxelarChain } from '@agoric/portfolio-api/src/constants.js';
+import type { CaipChainId } from '@agoric/orchestration';
+import type { Passable } from '@endo/marshal';
+import type { GMPAccountInfo } from './portfolio.exo.ts';
+
+const trace = makeTracer('EvmAccountKit');
+
+const nowIso = () => new Date().toISOString();
```

Comment:
That would need ambient access to the clock. It's not available in [contract compartments](https://docs.agoric.com/guides/js-programming/hardened-js.html#hardening-javascript-limiting-globals-with-compartments).